lib/Crypto/extensionplugins.h

```diff
@@ -44,6 +44,8 @@ class SAILFISH_CRYPTO_API CryptoPlugin
     CryptoPlugin();
     virtual ~CryptoPlugin();
 
+    virtual bool initialise(const QVariantMap &parameters) = 0;
```

Comment:
initiali**z**e()

Reply:
@chriadam one guy told me that it's correctly in Australia? I mean 's'.

Reply:
Yes, Australian English uses ise in such word-endings consistently.  British English is inconsistent between ise and ize, while American English consistently uses ize.  I will ask Joona which style we should prefer here, it may be that American English is preferred...

Reply:
@chriadam Thanks for explanation. I think it's up to @vityafx